cypress/e2e/stellar-wallet.cy.ts

```diff
@@ -4,51 +4,155 @@ beforeEach(() => {
   cy.visit(Cypress.env('base_url'));
 });
 
-describe('UI Layout', () => {
-  describe('Navbar', () => {
-    it('should show a navbar', () => {
-      cy.get('[data-cy="nav-container"]').should('exist').and('be.visible');
-      cy.get('[data-cy="nav-logo"]').should('be.visible').and('have.attr', 'alt', 'Stellar Logo');
-      cy.get('[data-cy="nav-logo-link"]')
-        .should('be.visible')
-        .and('have.attr', 'href', Cypress.env('URL_STELLAR'));
+describe('Stellar Wallet management', () => {
+  describe('UI Layout', () => {
+    describe('Navbar', () => {
+      it('should show a navbar', () => {
```

Comment:
Capitalize every Should inside our it please, that way we keep consistency in our code.